addon/services/body-class.js

```diff
@@ -51,6 +52,10 @@ export default class BodyClassService extends Service {
   }
 
   willDestroy() {
-    cancel(this.scheduledRun);
+    if (this._fastboot && this._fastboot.isFastBoot) {
```

Comment:
How does it exactly "prevent FastBoot from removing the CSS classes again"?

We'll go through this loop every times we come from FastBoot, even if it's not the first one. Or am I missing something?

Reply:
> How does it exactly "prevent FastBoot from removing the CSS classes again"?

by canceling the final `updateBodyClass()` call here, which would remove all dynamic CSS classes from the body element.

> We'll go through this loop every times we come from FastBoot, even if it's not the first one. Or am I missing something?

I not sure which loop you are talking about 🤔

Reply:
> by canceling the final updateBodyClass() call here, which would remove all dynamic CSS classes from the body element.

But wasn't it already the case?
It seems that we're **now** doing the cancellation only when `isFastBoot` VS doing it everytime **before**, am I wrong?

Reply:
> It seems that we're now doing the cancellation only when isFastBoot VS doing it everytime before, am I wrong?

nope, that is exactly correct and matches the PR title 😉